src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/regler/vilkår/PassBarnRegel.kt

```diff
@@ -68,4 +115,14 @@ class PassBarnRegel : Vilkårsregel(
                 ),
             )
     }
+
+    fun harFullførtFjerdetrinn(fødselsdato: LocalDate, datoForBeregning: LocalDate = LocalDate.now()): Boolean {
```

Comment:
Denne får jeg ta en titt på imorgen 👀

Reply:
Denne burde kanskje legges i en egen fil, og testes separat 👀 
Hvis jeg tolker kommentarene riktig så er det sånn at:
* Man har fullført fjerdetrinn det det år man fyller 10 år, uavhengig dato i året man fyller.

```
2013 født
2014 fyller 1 år
2015 fyller 2 år
2016 fyller 3 år
2017 fyller 4 år
2018 fyller 5 år
2019 fyller 6 år - starter førsteklass
2020 fyller 7 år - fullfører 1 klass
2021 fyller 8 år - fullfører 2 klass
2022 fyller 9 år - fullfører 3 klass
2023 fyller 10 år - fullfører 4 klass
```

Jeg tenker man vanligvis avslutter ett trinn en gang i juni. Hvis datoet for beregning er eks i 2023
* Hvis datoForBeregning er før juni, så må barnet avsluttet fjerdetrinn forrige år
* Hvis datoForBeregning er etter juni, så har man avsluttet trinnet i juni, og då må være over 9 år
```kotlin
    return if (datoForBeregning.month >= Month.JUNE) {
        datoForBeregning.year - fødselsdato.year > 9
    } else {
        datoForBeregning.year - fødselsdato.year > 10
    }
```

Nesten den samme logikken, men enda ikke 👀 
Forskjellen i resultat blir når man sjekker `harFullførtFjerdetrinn` i mai:
```
fødselsår=2013 datoForBeregning=2023-05-01
fødselsdato=2013-01-01 datoForBeregning=2023-05-01 før=true etter=false
fødselsdato=2013-05-01 datoForBeregning=2023-05-01 før=true etter=false
fødselsdato=2013-08-01 datoForBeregning=2023-05-01 før=true etter=false
fødselsdato=2013-12-01 datoForBeregning=2023-05-01 før=true etter=false
```

Reply:
Finnes det de som starter i førsteklass året før/etter? Burde vi ha sikkerhetsmarginal på 1 år?